examples/any-type/tests/any_type_test.bal

```diff
@@ -10,9 +10,11 @@ string[] outputs = [];
 test:MockFunction mock_printLn = new();
 
 public function mockPrint(any|error... val) {
-    outputs.push(val.reduce(function (any|error a, any|error b) returns string => a.toString() + b.toString(), "").toString());
+    outputs.push(toString(val.reduce(function (any|error a, any|error b) returns string => toString(a) + toString(b), "")));
```

Comment:
Line longer than 80 characters? This will fail at the `LengthTest`. Shall we reduce the length to 80 chars?

Reply:
Oh this is a test. Ignore the comment